modules/post/linux/gather/enum_protections.rb

```diff
@@ -129,24 +132,130 @@ def check_hardening
     end
   end
 
-  def find_apps
-    apps = %w[
```

Comment:
Is there a reason some of these may have been dropped? I just noticed `truecrypt` has been dropped for example.

Reply:
Err, this was supposed to got into a separate commit, but it seems that I stashed everything. The reasoning was that the removed tools aren't really protection-related in any way.

Reply:
truecrypt, webmin, and pwgen were removed. I'm with @jvoisin on this one- none of these would likely be something that would be considered a "protection" from an interactive session.
Truecrypt is the only one that may be useful (and that's a big stretch), but that has not been updated in a decade so I don't think it should be included anymore.